client/src/includes/contentMetrics.ts

```diff
@@ -0,0 +1,86 @@
+interface ContentMetrics {
+  wordCount: number;
+  readingTime: number;
+}
+
+interface SegmentData {
+  segment: string;
+  isWordLike?: boolean | undefined;
+}
```

Comment:
We shouldn’t need to define this ourselves since it’s a built-in API. Looking at the TypeScript source, it seems to be [part of` ES2022.intl`](https://github.com/microsoft/TypeScript/blob/main/src/lib/es2022.intl.d.ts). See if you can update our `tsconfig.json` to include this?